src/api/inventory_api.rs

```diff
@@ -42,12 +42,7 @@ impl InventoryApi {
         catalog_object_id: &str,
         params: RetrieveInventoryCountParams,
     ) -> Result<RetrieveInventoryCountResponse, ApiError> {
-        let url = format!(
-            "{}/{}{}",
-            &self.url(),
-            catalog_object_id,
-            params.to_query_string()
-        );
+        let url = format!("{}/{}{}", &self.url(), catalog_object_id, params.to_query_string());
```

Comment:
I always think it's better to have this add extra /'s and then remove doubles, so that people don't have to remember if they need a trailing slash on self.url or not.

Reply:
The second one is for the query string.  I could put a "?" but it would be weird if there doesn't end up being any parameters.

Reply:
I see...but this would break if I set the url as:
https://api.square.com/

Reply:
because it would have https://api.square.com//something